marketplace-ui/src/app/modules/release-preview/release-preview.component.scss

```diff
@@ -0,0 +1,249 @@
+/* General Styles */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+/* Container */
+.container {
+  max-width: 100%;
+  margin: 0 auto;
+  padding: 20px;
+  border-radius: 8px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  margin-top: 40px;
+}
+
+.header {
+  text-align: center;
+  padding: 20px 0;
+
+  h2 {
+    text-align: center;
+    font-size: 4rem;
+    margin-bottom: 20px;
+  }
+}
+
+.hint-text {
+  font-size: 8px;
+}
+
+/* Form Styles */
+.form-group {
+  font-size: 18px;
+  margin-bottom: 15px;
+}
+
+/* File Input Styles */
+#fileInput {
+  border: 1px solid #ced4da;
+  padding: 10px;
+  width: 100%;
+  border-radius: 5px;
+  margin-top: 10px;
+}
+
+#fileInput:focus {
+  outline: none;
+  border-color: #5b9bd5;
+}
+
+/* Button Styles */
+button {
+  width: 10.1rem;
+  height: 3.5rem;
+  font-size: 1.6rem;
+  background-color: #007bff;
+  border: none;
+  color: white;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+}
+
+button:disabled {
+  cursor: not-allowed;
+}
+
+.no-tabs {
+  text-align: center;
+  padding: 50px;
+  font-size: 1.2rem;
+  color: #6c757d;
```

Comment:
please check color var of no-tabs, its not always #6c757d

Reply:
I think we should adapt var color same as rating-empty-text which is var(--text-no-rating-color)